crates/holochain/tests/tests/mod.rs

```diff
@@ -21,6 +21,8 @@ mod new_lair;
 mod paths;
 mod publish;
 mod regression;
+#[cfg(feature = "unstable-functions")]
+mod schedule;
```

Comment:
This is a good place to add tests like these. There are a lot of sweettests mixed into the codebase where ideally only unit tests would be and it makes it slow to run just the unit tests on Holochain. I'd really like to move all our integration tests out here.

Reply:
Does that work for you if I move the pre-existing schedule tests in that file in a subsequent PR?